includes/rest-api.php

```diff
@@ -149,15 +151,17 @@ function register_rest_routes() {
 		'distributor/list-pull-content',
 		array(
 			'methods'             => 'POST',
-			'callback'            => __NAMESPACE__ . '\get_pull_content',
+			'callback'            => __NAMESPACE__ . '\get_pull_content_list',
 			'permission_callback' => __NAMESPACE__ . '\\get_pull_content_permissions',
 			'args'                => get_pull_content_list_args(),
 		)
 	);
 }
 
 /**
- * Set the accepted arguments for the pull content list endpoint
+ * Set the accepted arguments for the pull content list endpoint.
+ *
+ * @since x.x.x Introduced the include, order and orderby arguments.
```

Comment:
We can probably set this to 2.0.0 I think

Reply:
Placeholders replaced in 9dd57461fd98b2067c5f3e5ea051f9180edb6b24